spec/rubocop/cop/layout/comment_indentation_spec.rb

```diff
@@ -292,4 +292,55 @@ def compile_sequence(seq, seq_var)
       RUBY
     end
   end
+
+  context 'when `Layout/AccessModifierIndentation EnforcedStyle: outdent`' do
+    let(:allow_for_alignment) { true }
```

Comment:
Looks like this is not needed, because you redefine `config` in this block (the single place where it was used).